File: Python/UberVan.py
from car import Car

class UberVan(Car):
    typeCarAcepted = []
    seatsMaterial = []

    def __init__(self, license, driver, typeCarAcepted, seatsMaterial):
        super().__init__(license, driver)
        self.typeCarAcepted = typeCarAcepted
        self.seatsMaterial = seatsMaterial

File: java/UberVan.java
import java.util.Map;
import java.util.ArrayList;

class UberVan extends Car{
    Map<String, Map<String, Integer>>  typeCarAcepted;
    ArrayList<String> seatsMaterial;
    private Integer passanger;

    public UberVan(String license, Account driver){
        super(license, driver);
      
    }

    @Override
    public void setPassager(Integer passager) {
        if(passager == 6){
            this.passanger = passager;
        }else{
        System.out.println("necesitas asignar 4 pasajeros");}
        
    }
}
